src/components/Navigation.js

```diff
@@ -21,6 +23,9 @@ import { useSelector } from 'react-redux';
  */
 function Navigation() {
   const useAtomicSwap = useSelector(state => state.useAtomicSwap);
+  const featureToggles = useSelector(state => state.featureToggles);
```

Comment:
thought(non-blocking): Should we integrate the atomic swap flag into the feature toggles in the future? This is a bit of information that also comes from unleash, so it seems fitting to have them all together.

Reply:
Issue created https://github.com/HathorNetwork/hathor-wallet/issues/604